DED-298 introducing additional changes to docs connected with HelpDesk (#1226)
livechat/livechat-public-docs: src/pages/getting-started/index.mdx

# Welcome to LiveChat

If you're wondering whether LiveChat would suit your business, this is the place for you to get started. We'd like to tell you about our products, their main functionalities as well as their high potential to be customized. Let's take you on a small tour!

> # What LiveChat is

Author: No właśnie miałam rozkminke jak to najlepiej ugryźć...zmienię bo w sumie Twoje zdanie przeważa szalę na to żeby zmienic xd i zobaczymy jeszcze co Ola powie

> # What LiveChat is

Reviewer: :D wpadłam też na pomysł teraz że może to "Welcome to LiveChat" zmienić na "Welcome to LiveChat Platform" lub coś podobnego żeby mieć takie szersze spojrzenie?

> # What LiveChat is

Author: Oooo to dobry pomysł!

- monetization tools and <Placeholder id="DEVELOPER_CONSOLE_URL"/>
- the <Placeholder id="LIVECHAT_MARKETPLACE_URL"/> of apps and services

# What HelpDesk is

[HelpDesk](https://www.helpdesk.com/) is a ticketing system that helps you to manage both your asynchronous and synchronous communication with customers. You can imagine it as a mailbox with many features that make communication much more manageable:

> - [Canned responses](https://www.helpdesk.com/help/what-are-canned-responses/)

Reviewer 2: Since these are all external links, we should use the `<a>` tag with "target blank", so that it opens in a new tab

- <a href="https://www.helpdesk.com/help/what-are-statuses/" target="_blank">Statuses</a>
- <a href="https://www.helpdesk.com/help/how-to-use-private-notes/" target="_blank">Private notes</a>
- <a href="https://www.helpdesk.com/help/how-to-use-filters/" target="_blank">Filters</a> and <a href="https://www.helpdesk.com/help/what-are-tags-and-how-to-use-them/" target="_blank">tags</a>
- <a href="https://www.helpdesk.com/help/what-are-teams/" target="_blank">Teams</a>
  

> Read more about HelpDesk [here](https://www.helpdesk.com/help/what-is-helpdesk/), and if you want to start your experience, check out  [3 simple steps showing how to begin](https://www.helpdesk.com/help/starting-with-helpdesk/). 

Reviewer 2: also external links

# Platform APIs

Many LiveChat and HelpDesk applications built upon the LiveChat Platform need to use our API. For that reason, we make it accessible and let developers play with it.

There are two **messaging APIs** for LiveChat:

- [Agent Chat API](/messaging/agent-chat-api/)
- [Customer Chat API](/messaging/customer-chat-api/)

There's also the [Configuration API](/management/configuration-api/).

> Public APIs for HelpDesk are widely described in the [HelpDesk API Documentation](https://api.helpdesk.com/docs#section/Introduction).

Reviewer 2: It's actually just one API

One of the most extensively used section in the Agent App is [Reports](https://www.livechat.com/help/reporting-options-available-in-livechat/). We provide several types of reports, which help managers with the analysis. You can measure chat satisfaction and duration, keep track of agents' activity, or compare time periods to observe trends. Again, this is just a tiny bit of what's available in terms of reporting. Yet, we're aware that what we offer does not cover every single business case. This is where [Reports API](/data-reporting/reports-api/) comes into play. It serves as a sets of tools to create apps which interact on the server side. With this API, it's possible to design exactly what you need.

# What HelpDesk is

> [HelpDesk](https://www.helpdesk.com/) is a ticketing system that helps you to manage your asynchronous and synchronous communication with customers. You can imagine it as a mailbox with many features that make communication much more manageable:

Reviewer: ```suggestion
[HelpDesk](https://www.helpdesk.com/) is a ticketing system that helps you to manage both your asynchronous and synchronous communication with customers. You can imagine it as a mailbox with many features that make communication much more manageable:
```

- [Canned responses](https://www.helpdesk.com/help/what-are-canned-responses/)
- [Statuses](https://www.helpdesk.com/help/what-are-statuses/)
- [Private notes](https://www.helpdesk.com/help/how-to-use-private-notes/)
- [Filters](https://www.helpdesk.com/help/how-to-use-filters/) and [tags](https://www.helpdesk.com/help/what-are-tags-and-how-to-use-them/) 
- [Teams](https://www.helpdesk.com/help/what-are-teams/) 
  
Read more about HelpDesk [here](https://www.helpdesk.com/help/what-is-helpdesk/), and if you want to start your experience, check out  [3 simple steps showing how to begin](https://www.helpdesk.com/help/starting-with-helpdesk/). 

# HelpDesk and LiveChat integration

> You can integrate LiveChat with HelpDesk to get a powerful tool helping you to manage all your communication with the clients in one place. 

Reviewer: ```suggestion
You can integrate LiveChat with HelpDesk to get a powerful tool helping you to manage all your communication with your customers in one place. 
```

Reviewer: w komunikacj raczej nie używamy raczej "clients" mówiąc o customerach 😌

> [Read more](https://www.helpdesk.com/help/livechat-integration-manage-tickets-and-chats-in-one-application/) about HelpDesk and Livechat integration, or buy our service directly on our [Marketplace](https://www.livechat.com/marketplace/apps/helpdesk/#description).

Reviewer: ```suggestion
[Read more](https://www.helpdesk.com/help/livechat-integration-manage-tickets-and-chats-in-one-application/) about HelpDesk and LiveChat integration or buy our service directly on the [Marketplace](https://www.livechat.com/marketplace/apps/helpdesk/#description).
```
